packages/eas-cli/src/build/utils/resourceClass.ts

```diff
@@ -8,24 +8,30 @@ import { BuildResourceClass } from '../../graphql/generated';
 import Log from '../../log';
 import { getReactNativeVersionAsync } from '../metadata';
 
+type AndroidResourceClass = Exclude<
```

Comment:
Maybe `Pick<>` is better than `Exclude<>`? If we ever add an iOS resource class, you'll have to remember about updating this type.

Reply:
I think you can't do `Pick` on an enum in TS 🤔. Or maybe I'm wrong?